### ISSUE TITLE
[Feature Request] expansion of fossbilling API to  incorporate commands which facilitate addition/deletion of TLDs and editing of TLD pricing   

### ISSUE THREAD

**Issue by DarthCruiser** [FOSSBilling/FOSSBilling#2075]
Requested features - 
 API command(s) to programatically:
 - add/delete TLDs from fossbilling
 - edit live TLD prices 
 - enhance fossbillig functionality to enable setting of registration/renewal periods to match those available from the registrar - for example:
1 year
1 - 5 years
1-10 years
2-10 years
1 , 2 or 5 years
1, 2, 3 or 5 years
1, 2, 3, 4, 5 or 10 years
The above is just a few examples from the Netim - API response for PeriodCreate and PeriodRenew.

**Comment by ZaDeveloper** [comment]
Yes this important feature... and must support the popular registrars

**Comment by admdly** [comment]
This was partially implemented in the past, and I've gone ahead and completed the remaining part (in #4115) to fully resolve this. The changes will be in the latest preview soon (allow ~60mins for it to update) and in the next version due out soon (0.8.6).
